client/src/modals/PaymentModal.js

```diff
@@ -0,0 +1,49 @@
+import '../styles/Modals.css';
+import '../styles/HomePage.css';
+import React from 'react';
+import Button from "@mui/material/Button";
+import Modal from '@mui/material/Modal';
+import Box from '@mui/material/Box';
+import ConfirmationNumberIcon from '@mui/icons-material/ConfirmationNumber';
+
+function EditEventModal(props)
```

Comment:
Change name to PaymentModal :D It works like this because this file has a default export so you can name it anything when you import it, but would still porbably be nice to call this what it actually is 🤪